src/domains.js

```diff
@@ -37,7 +37,9 @@ export function buildPayPalAPIUrl(path : string = '') : string {
 }
 
 export function getPayPalLoggerUrl() : string {
-    return buildPayPalUrl(URI.LOGGER);
+    return __FORCE_PAYPAL_DOMAIN_LOGGER__ ?
```

Comment:
Modified code here to not affect the logic on `buildPayPalUrl`. Not sure if do we need to specialize the code a little more.